go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/cloudwego/netpoll-http2 v0.0.6
 	github.com/cloudwego/thriftgo v0.1.2
 	github.com/json-iterator/go v1.1.11
+	github.com/stretchr/testify v1.7.0
```

Comment:
There is an assertion library `internal/test` shipped with kitex widely used in kitex's unit tests. So do not introduce any third-party test library. Use `internal/test` instead.

Reply:
The internal/test library does not have mock interface capabilities, only assert capabilities.
In my submission, I can replace assert with this library. However, the mock file generated by Mock Gen still introduces this dependency. Do you think it's ok?